docs/config.md

```diff
@@ -55,6 +55,35 @@ for more details.
 | `HttpClient` | [System.Net.Http.HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient) and [System.Net.HttpWebRequest](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest) | * | source |
 | `NetRuntime` | [OpenTelemetry.Instrumentation.Runtime](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Runtime) | * | source |
 
+## Logs
+
+Logs from ASP.NET Core apps could be captured using two options. First option is
+to use HostingStartup and the second option is to rely on CLR Profiler
+Instrumentation. Recommended option for ASP.NET Core is to use HostingStartup.
+For other .NET Core apps logs could be collected only through .NET CLR Profiler
+Instrumentation.
```

Comment:
```suggestion
Logs from ASP.NET Core apps can be captured in two ways: The first option is
to use `HostingStartup`, and the second is to rely on CLR Profiler Instrumentation. 
For ASP.NET Core, use `HostingStartup`. For other .NET Core apps, you can only
collect logs through .NET CLR Profiler Instrumentation.
```

Reply:
If both are present `HostingStartup` adds the logger integration. There is no harm in having both of them.